packages/flutter_test/test/accessibility_test.dart

```diff
@@ -43,6 +43,38 @@ void main() {
       handle.dispose();
     });
 
+    testWidgets('White text on white background fails contrast test',
+        (WidgetTester tester) async {
+      final SemanticsHandle handle = tester.ensureSemantics();
+      await tester.pumpWidget(
+        _boilerplate(
+          Container(
+            width: 200.0,
+            height: 200.0,
+            color: Colors.white,
+            child: Column(
+              children: const <Widget>[
+                Text(
+                  'this is a white text',
+                  style: TextStyle(fontSize: 14.0, color: Colors.white),
+                ),
+                Text(
+                  'this is a black text test1',
+                  style: TextStyle(fontSize: 14.0, color: Colors.black),
+                ),
+                Text(
+                  'this is a black text test2',
```

Comment:
Yeah, this is the problem I was mentioning - we need to figure out why this is not tripping up the guideline test